ci-operator/step-registry/build/github-secrets/build-github-secrets-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
+  as: build-github-secrets
+  from: cli
+  commands: build-github-secrets-commands.sh
+  resources:
+    requests:
+      cpu: 50m
+      memory: 100Mi
+  credentials:
+  - namespace: test-credentials
```

Comment:
so the `test-credentials` namespace and these 2 secrets already exist on the CI cluster, but we just need to copy them to the `build-e2e-github-secrets` namespace @adambkaplan ?

where I'm guessing it is ok for our e2e's to access `build-e2e-github-secrets` but not `test-credentials` during the actual test runs ... that sound right?

Reply:
`test-credentials` is a namespace on our build farm cluster (api.ci, build01, or build02). The `openshift-tests` binary does not have permission to read secrets on our build farm - it gets access to secrets via build mounts.

When the steps run, they create a kubeconfig file for the cluster under test, and its location is saved to the KUBECONFIG environment variable. Therefore, the `build-e2e-github-secrets` namespace exists in the _cluster being tested_, which the `openshift-tests` binary can access with full admin privileges.

Reply:
Docs for adding secrets in CI:

https://docs.ci.openshift.org/docs/how-tos/adding-a-new-secret-to-ci/

Reply:
thanks for the detail @adambkaplan 

so "yes" to my questions :-)

/lgtm